e2e/tests/stablex_test.rs

```diff
@@ -202,12 +202,5 @@ fn test_rinkeby() {
     println!("Sleeping {} seconds...", sleep_time);
     std::thread::sleep(Duration::from_secs(sleep_time));
 
-    // Make sure there was no error
-    let output = Command::new("docker-compose")
-        .arg("logs")
-        .output()
-        .expect("failed to execute process");
-    // Errors go to stderr while other messages go to stdout.
-    let logs = String::from_utf8(output.stderr).expect("failed to read logs");
-    assert!(logs.is_empty());
+    docker_logs::assert_no_errors_logged();
```

Comment:
maybe we can make this more generic by passing in the Docker compose service name (here `"stablex"`). The reason I say this is that the name `assert_no_errors_logged` doesn't capture that this is JUST for the `stablex` service.